ckanext/b2find/controller.py

```diff
@@ -16,3 +16,44 @@ def nutzungsbedingungen(self):
 class DocsController(base.BaseController):
```

Comment:
Please remove this class, move the file to help/ and use HelpController. It would be more consistent. Remember to update "map.connect..." and the HTML.